app/views/layouts/application_revised_bikehub.html.haml

```diff
@@ -8,39 +8,41 @@
     <link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
     = javascript_include_tag 'application_revised'
     <!--[if IE]>$('body').prepend("<div id='old-browser-warning'><h4>Your browser is out of date!</h4><p>As a result, Bike Index will not function correctly. <a href=\"http://whatbrowser.com\">Learn more here</a>.</p></div>")<![endif]-->
+
   %body{ id: page_id, class:  body_class, style: "padding-top: 0;" }
-    / This is either the signup or signin page
+    -# This is either the signup or signin page
     - signup_page = controller_name != "sessions"
     %nav.navbar.nav.bikehub-nav
       .container-fluid
         .row
           %a.bikehub-logo{ href: "https://bikehub.com" }
             = image_tag "logos/bikehub_logo.png", class: "primary-nav", alt: "BikeHub home"
           .poweredby
-            %p
-              powered by
+            %p= t(".powered_by")
             %a.primary-logo{ href: user_root_url }
               = image_tag "logo.svg", class: "primary-nav", alt: "Bike Index home"
           %a.bikehub-nav-item.bikehub-nav-item-first{ href: "https://bikehub.com/contact/" }
-            Contact
+            = t(".contact")
           %a.bikehub-nav-item{ href: new_user_path(partner: sign_in_partner), class: (signup_page ? "active" : "") }
-            Sign up
+            = t(".sign_up")
           %a.bikehub-nav-item{ href: new_session_path(partner: sign_in_partner), class: (signup_page ? "" : "active") }
-            Log in
+            = t(".log_in")
     .container.bikehub-explanation.mb-4
       .row
         .col-sm-12
           %h1
-            BikeHub
-            = signup_page ? "Sign up" : "Log In"
+            = t(".bikehub")
```

Comment:
Proper name, shouldn't be translated.

Reply:
That's a useful smell -- generally it's a good idea to have entries be complete semantic units as much as possible